<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>bballConfig</class>
 <widget class="QWidget" name="bballConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>546</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configure BbalL!</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="3">
    <widget class="QLabel" name="label_5">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Appearance</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2" colspan="2">
    <widget class="KUrlRequester" name="imageUrl">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="filter">
      <string>*.png *.gif *.svg *.svgz *.jpeg *.jpg</string>
     </property>
     <property name="fileDialogModality">
      <enum>Qt::NonModal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="2" colspan="2">
    <layout class="QHBoxLayout">
     <item>
      <widget class="KColorButton" name="colourize">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="colourizeEnabled">
       <property name="text">
        <string>Enabled</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="colourizeOpacityLabel">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Colorize opacity:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>colourizeOpacitySlider</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="2" colspan="2">
    <widget class="QSlider" name="colourizeOpacitySlider">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QLabel" name="label_6">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Physics</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2" colspan="2">
    <widget class="QSlider" name="gravity">
     <property name="maximum">
      <number>500</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="6" column="2" colspan="2">
    <widget class="QSlider" name="friction">
     <property name="maximum">
      <number>30</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="7" column="2" colspan="2">
    <widget class="QSlider" name="resitution">
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Sound enabled:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>soundEnabled</cstring>
     </property>
    </widget>
   </item>
   <item row="10" column="2">
    <widget class="QCheckBox" name="soundEnabled">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="11" column="2" colspan="2">
    <widget class="QSlider" name="soundVolume">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="12" column="1">
    <widget class="QLabel" name="soundFileLabel">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Bounce effect:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>soundFile</cstring>
     </property>
    </widget>
   </item>
   <item row="12" column="2" colspan="2">
    <widget class="KUrlRequester" name="soundFile">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="13" column="0" colspan="2">
    <widget class="QLabel" name="label_9">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Auto Bounce</string>
     </property>
    </widget>
   </item>
   <item row="18" column="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>361</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Image:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>imageUrl</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="colourizeLabel">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Colorize:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>colourize</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Gravity:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>gravity</cstring>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Friction:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>friction</cstring>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Restitution:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>resitution</cstring>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="3">
    <widget class="QLabel" name="label_8">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Sound</string>
     </property>
    </widget>
   </item>
   <item row="11" column="1">
    <widget class="QLabel" name="soundVolumeLabel">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Volume:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>soundVolume</cstring>
     </property>
    </widget>
   </item>
   <item row="15" column="2">
    <widget class="QCheckBox" name="autoBounceEnabled">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="15" column="1">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>Auto bounce:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>autoBounceEnabled</cstring>
     </property>
    </widget>
   </item>
   <item row="17" column="1">
    <widget class="QLabel" name="autoBounceStrengthLabel">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Auto bounce strength:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>autoBounceStrength</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="17" column="2">
    <widget class="QSlider" name="autoBounceStrength">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="maximum">
      <number>50</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KColorButton</class>
   <extends>QPushButton</extends>
   <header>kcolorbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>colourizeEnabled</sender>
   <signal>clicked(bool)</signal>
   <receiver>colourize</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>302</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>218</x>
     <y>160</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>colourizeEnabled</sender>
   <signal>clicked(bool)</signal>
   <receiver>colourizeOpacitySlider</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>372</x>
     <y>160</y>
    </hint>
    <hint type="destinationlabel">
     <x>375</x>
     <y>230</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>autoBounceEnabled</sender>
   <signal>clicked(bool)</signal>
   <receiver>autoBounceStrength</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>177</x>
     <y>104</y>
    </hint>
    <hint type="destinationlabel">
     <x>182</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>autoBounceEnabled</sender>
   <signal>clicked(bool)</signal>
   <receiver>autoBounceStrengthLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>212</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>80</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>colourizeEnabled</sender>
   <signal>clicked(bool)</signal>
   <receiver>colourizeLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>340</x>
     <y>145</y>
    </hint>
    <hint type="destinationlabel">
     <x>56</x>
     <y>163</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>colourizeEnabled</sender>
   <signal>clicked(bool)</signal>
   <receiver>colourizeOpacityLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>404</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>92</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>soundEnabled</sender>
   <signal>clicked(bool)</signal>
   <receiver>soundVolumeLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>130</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel">
     <x>63</x>
     <y>157</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>soundEnabled</sender>
   <signal>clicked(bool)</signal>
   <receiver>soundVolume</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>284</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel">
     <x>289</x>
     <y>151</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>soundEnabled</sender>
   <signal>clicked(bool)</signal>
   <receiver>soundFileLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>233</x>
     <y>99</y>
    </hint>
    <hint type="destinationlabel">
     <x>55</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>soundEnabled</sender>
   <signal>clicked(bool)</signal>
   <receiver>soundFile</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>253</x>
     <y>106</y>
    </hint>
    <hint type="destinationlabel">
     <x>269</x>
     <y>221</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
